Documentation/project-docs/developer-guide.md

```diff
@@ -321,3 +321,7 @@ The CoreFX build and test suite is a work in progress, as are the [building and
     5) Run the tests by any means you please - the binary shouldn't get overwritten.
 
 If you prefer, you can use a Debug build of System.Private.CoreLib, but if you do you must also use a Debug build of the native portions of the runtime, e.g. coreclr.dll. Tests with a debug runtime will execute much more slowly than with Release runtime bits.
+
+To collect code coverage that includes types in System.Private.CoreLib.dll, you'll need to follow the above steps, then
+
+`msbuild /t:rebuildandtest /p:Coverage=true /p:CodeCoverageAssemblies="System.Private.CoreLib" /p:ConfigurationGroup=Release`
```

Comment:
> It depends. I built everything using the release flag,including Corefx, so when I tried to build without /p:ConfigurationGroup=Release, I got an error because of missing files in folder Debug/.... .

To be clear, /p:ConfigurationGroup=Release had nothing to do with what coreclr is used, which is why I said it isn't necessary. The coreclr used will be whichever you copied the directory. This flag controls which corefx is used.